loader/loader.go

```diff
@@ -503,7 +511,7 @@ func newResult() *Result {
 }
 
 func all(paths []string, filter Filter, f func(*Result, string, int) error) (*Result, error) {
-	errs := Errors{}
+	errors := Errors{}
```

Comment:
[nit] looks like you've undone a bit more than just the deletion of `AsBundle`. But it's ok 😄